connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -2985,7 +3278,24 @@ void Gap::on_remote_connection_parameter(
 #if BLE_ROLE_OBSERVER
 ble_error_t Gap::setScanParameters(const ScanParameters &params)
 {
+    tr_info("Set scan parameters - "
+            "own_address_type=%s, "
+            "scanning_filter_policy=%s, "
+            "phys=%d ,"
+            "phy_1m_configuration:[interval=%lums,window=%lums,active_scanning=%s] ,"
+            "phy_coded_configuration:[interval=%lums,window=%lums,active_scanning=%s]",
+            to_string(params.getOwnAddressType()),
+            to_string(params.getFilter()),
+            params.getPhys().value(),
+            params.get1mPhyConfiguration().getInterval().valueInMs(),
+            params.get1mPhyConfiguration().getWindow().valueInMs(),
+            to_string(params.get1mPhyConfiguration().isActiveScanningSet()),
+            params.getCodedPhyConfiguration().getInterval().valueInMs(),
+            params.getCodedPhyConfiguration().getWindow().valueInMs(),
+            to_string(params.getCodedPhyConfiguration().isActiveScanningSet()));
```

Comment:
should be dependent on the phy selected

Reply:
Can you please elaborate on this?

Reply:
why are you printing coded phy configuration if it's not set?

Reply:
OK (see c305e77)